public_html/wp-content/plugins/wordcamp-forms-to-drafts/wordcamp-forms-to-drafts.php

```diff
@@ -378,11 +380,19 @@ protected function get_speaker_from_user_id( $user_id ) {
 	 * @return int | WP_Error
 	 */
 	protected function create_draft_speaker( $speaker ) {
+		$content = ( ! empty( $speaker['Your Bio'] ) ) ? $speaker['Your Bio'] : '';
```

Comment:
Could this be shortened to `$speaker['Your Bio'] ?? ''` ?

If not, I don't think the parens are needed around the condition, and to me it feels like it makes it harder to read. Not a big deal either way though.

Same applies to below code too.

Reply:
Ah good point. I always get `??` and `?:` confused, and `?:` wasn't working (see previous commit) so I just switched to the longform ternary. This `??` seems to work, though.